cartography/intel/aws/redshift.py

```diff
@@ -0,0 +1,118 @@
+import logging
+
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get_redshift_cluster_data(boto3_session, region):
+    client = boto3_session.client('redshift', region_name=region)
```

Comment:
client can be a global var so you don't need to initialize upon each call.

If you don't want to initialize on load, make this a singleton

```
_client = None

def _client():
    global _client
    if _client is None:
        _client = boto3_session.client('redshift', region_name=region)
    return _client
```

Reply:
Good point, I'll keep this in consideration if we decide to make these sync scripts more generalized since there's a lot of boilerplate that I'm copying over and over. Right now though I'm going to keep this part as-is since it's the same as the other sync scripts.